### PR TITLE
Bump clustertest to v0.2.0

### PR DESCRIPTION
**Pull request by fiunchinho**
### What this PR does
Upgrade `clustertest` to its latest release.

### Checklist

- [ ] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

**Comment by fiunchinho** [comment]
@giantswarm/team-tinkerers @giantswarm/team-rocket do you know if CAPV tests are supposed to pass?

**Comment by AverageMarcus** [comment]
As far as I know they should be. They passed yesterday - https://github.com/giantswarm/cluster-test-suites/pull/66

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Bumped clustertest to `v0.1.1` to fix `GITHUB_TOKEN` issue
+- Bumped clustertest to `v0.1.1` to fix `GITHUB_TOKEN` issue.
+- Bumped clustertest to `v0.2.0` to use custom app values.
```
Comment (AverageMarcus) [inline CHANGELOG.md:34]: This needs moving up to the unreleased section

Comment (fiunchinho) [reply CHANGELOG.md:38]: Fixed

**Comment by fiunchinho** [comment]
> As far as I know they should be. They passed yesterday - #66

I'm investigating a bit. I think tests failed because there was no `default-apps` app on CAPV.

**Review by AverageMarcus — APPROVED** [review]

**Review by mnitchev — APPROVED** [review]

**Pull request merged**